Upstream-Status: pending

Automake 1.12 has deprecated automatic de-ANSI-fication support

This patch avoids this issue with automake 1.12:

| configure.in:22: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02

Index: shadow-4.1.4.3/configure.in
===================================================================
--- shadow-4.1.4.3.orig/configure.in
+++ shadow-4.1.4.3/configure.in
@@ -19,7 +19,6 @@ AC_PROG_CC
 AC_ISC_POSIX
 AC_PROG_LN_S
 AC_PROG_YACC
-AM_C_PROTOTYPES
 AM_PROG_LIBTOOL
 
 dnl Checks for libraries.
